Malayalam/malayalam.md

```diff
@@ -12,3 +12,5 @@
 
 [ശശി] (../english/tintumon-mallu/tintu-main.md)
 
+[ടിന്റുവും മാഷും] (../malayalam/tintu/tintu-teacher.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.